### DISCUSSION TITLE
DIY ESPixelStick diagram check

### DISCUSSION THREAD

**Discussion by fedekrum** [forkineye/ESPixelStick#713] [General]
This is not a bug or a feature. This is a doubt.
I did the following and it just works. Awesome!!

![Captura de Pantalla 2023-12-18 a la(s) 2 35 25](https://github.com/forkineye/ESPixelStick/assets/4470683/46eb0f59-f539-4dce-8e2c-36082658f607)

In the diagram, there are question marks because:
1) I am not sure if I have to connect pin 1 of the XLR to the other pin not connected on the HW-519. It just works without the connection.

2) How many amps must the USB power supply have for this implementation?

3) Is it OK to power it with a USB power supply (a charger)?

4) is there something missing or wrong in my diagram?

Thanks in advance

**Comment by MartinMueller2003** [comment]
RS485 only needs the two wires. The gnd wire is optional. It helps reduce noise on very long (300m+) runs. This configuration will draw about 500ma from the charger so a 5w (1amp) charger will work fine.

**Comment by fedekrum** [comment]
Thanks Martin.
Do you know of any virtual DMX controller on iPad or Android tablet able to send Arnet or Sacn?
I imagine that there must be something with sliders.

**Comment by MartinMueller2003** [comment]
I use sACNview on a windows PC. They may have a Mac version.

**Reply by fedekrum** [reply]
Thanks. I was looking for something to slide with my fingers (basically a tablet)

**Comment by fedekrum** [comment]
So this is better and complete for you based on your corrections?

![Captura de Pantalla 2023-12-19 a la(s) 0 01 28](https://github.com/forkineye/ESPixelStick/assets/4470683/bf1ea396-14d1-4b99-bd16-667c89af47bd)

Tested with the multimeter, both grounds on the HW-519 don't have continuity. Do you know if this is normal?

**Reply by MartinMueller2003** [reply]
Both GNDs? 
The diagram looks fine.

**Reply by fedekrum** [reply]
this two

![Captura de Pantalla 2023-12-19 a la(s) 9 53 17](https://github.com/forkineye/ESPixelStick/assets/4470683/4ef90dc3-8f10-4fc5-bd94-c05c974512b1)

should I connect them?

![291533206-5ced807e-095c-47e4-bdb0-a0b5ae99f135](https://github.com/forkineye/ESPixelStick/assets/4470683/c4f4f4a5-528c-4ca3-a62f-d43ae2e633de)

**Reply by MartinMueller2003** [reply]
No. Leave it the way you had it.

**Comment by fedekrum** [comment]
Thanks Martin.
